src/user.js

```diff
@@ -0,0 +1,15 @@
+import users from './users.json';
+
+export function seedUserData(){
+  var userModules = users.students[0].tracked_modules;
+  return hoodie.store.add({
+    "userModules": userModules
+  })
+  .then(() => {
+    console.log('your initial data has been given to hoodie');
+    return userModules;
```

Comment:
I believe in this case, it's not undefined... We're creating the Variable userModules already before, so it should exist independently from the Promise?